src/library-authoring/components/BaseComponentCard.tsx

```diff
@@ -26,7 +27,7 @@ type BaseComponentCardProps = {
 const BaseComponentCard = ({
```

Comment:
@rpenido I think we should rename this component to `BasicCard`, as it is no longer just used for library components. What do you think?

Reply:
Agreed! Fixed here: a863cf34e1a4ea365e991df82eddced649cd8e49